src/bundle/Security/Authentication/Exception/ReusedTwoFactorCodeException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scheb\TwoFactorBundle\Security\Authentication\Exception;
+
+use Symfony\Component\Security\Core\Exception\BadCredentialsException;
+
+/**
+ * @final
+ */
+class ReusedTwoFactorCodeException extends BadCredentialsException
+{
+    public const MESSAGE = 'Reused two-factor authentication code.';
+    private const MESSAGE_KEY = 'code_reused';
```

Comment:
The translation string should be added in `SchebTwoFactorBundle.en.yml`. And if you can add extra translations, would be much appreciated. I could provide a German translation, and I guess so could you ;)